L1Trigger/RPCTechnicalTrigger/src/RBCProcessRPCDigis.cc

```diff
@@ -327,34 +237,33 @@ int RBCProcessRPCDigis::getBarrelLayer( const int & _layer, const int & _station
 }
 
 
-void RBCProcessRPCDigis::setDigiAt( int sector, int digipos )
+void RBCProcessRPCDigis::setDigiAt( int sector, int digipos, RPCData& block )
 {
   
   int pos   = 0;
   int isAoB = 0;
 
   if ( m_debug ) std::cout << "setDigiAt" << std::endl;
   
-  std::vector<int>::const_iterator itr;
-  itr = std::find( m_sec1id.begin(), m_sec1id.end(), sector );
+  auto itr = std::find( s_sec1id.begin(), s_sec1id.end(), sector );
   
-  if ( itr == m_sec1id.end()) {
-    itr = std::find( m_sec2id.begin(), m_sec2id.end(), sector );
+  if ( itr == s_sec1id.end()) {
+    itr = std::find( s_sec2id.begin(), s_sec2id.end(), sector );
     isAoB = 1;
   } 
   
   for ( pos = 0; pos < 6; ++pos ) {
-    if (this->m_block->m_sec1[pos] == sector || this->m_block->m_sec2[pos] == sector )
```

Comment:
`this->m_block` is only being used as a _hidden_ argument to the function. I made it explicit.